custom_components/tuya_local/devices/cecotec_bigdry_4000_dehumidifier.yaml

```diff
@@ -0,0 +1,145 @@
+name: BigDry 4000 Expert
+products:
+  - id: d44nan1uokr1hlwf
+    name: Cecotec BigDry 4000 Expert
+    # {"dps": {"1": true, "3": "60", "4": "low", "6": 54, "7": 23, "19": 0}}
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  name: Dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 3
+      type: integer
+      name: humidity
+      range:
+        min: 40
+        max: 80
+    - id: 6
+      name: current_humidity
+      type: integer
+
+secondary_entities:
+  # This is twice to make it more prominent
+  - entity: switch
+    name: Power
+    dps:
+      - id: 1
+        type: boolean
+        name: switch
+  - entity: select
+    name: Humidity Target
+    dps:
+      - id: 3
+        type: string
+        name: option
+        mapping:
+          - dps_val: "40"
+            value: "40"
+          - dps_val: "45"
+            value: "45"
+          - dps_val: "50"
+            value: "50"
+          - dps_val: "55"
+            value: "55"
+          - dps_val: "60"
+            value: "60"
+          - dps_val: "65"
+            value: "65"
+          - dps_val: "70"
+            value: "70"
+          - dps_val: "75"
+            value: "75"
+          - dps_val: "80"
+            value: "80"
+  - entity: select
```

Comment:
Use a fan entity, or even the humidifier entity's mode for this.

Reply:
Thanks Jason for the hint. Indeed I decided to put it as extra selection because I do not like that wheel as selector and sometimes it was in read-only mode for no known reason.